web_widget_text_count/readme/ROADMAP.rst

```diff
@@ -0,0 +1,4 @@
+Patch Text class on Python level to allow propagation of size same as in Char fields.
+After modify js to relies it on field attribute.
```

Comment:
~After modify js to relies it on field attribute.~
"At the moment the `size` attribute can be applied only as an attribute on the `<field`/>` tag.

Reply:
@simahawk You can use the ``+-`` button ("Insert a suggestion") in the comments toolbar